badge/banner.php

```diff
@@ -83,7 +87,7 @@ function renderImage($textLines, $user = false) {
     $duration = sprintf(
         "%d days, %d:%02d hours",
         $row["duration"] / 1440,
-        ($row["duration"] / 60) % 24,
+        intdiv($row["duration"], 60) % 24,
```

Comment:
This looks a bit weird to be (I'd probably be more tempted to use `(int)` casting), but very much a stylistic choice. So doesn't really matter.

I know some IDE's complain about some calls like this for performance reasons, but this doesn't seem to be one of them.